extensions/interactions/base_test.py

```diff
@@ -203,6 +203,9 @@ def test_interaction_properties(self):
     def test_interaction_rules(self):
         """Tests the interaction rules."""
         def _check_num_interaction_rules(interaction_id, expected_num):
+            """Checks the number of rules in the interaction corresponding to
```

Comment:
Missing args section

Reply:
Done

Reply:
I thought you said test files didn't need args/returns/raises. Does the linter fail if the test files don't have all the sections of the docstring? If not, can you point me to the code that skips checking these sections on test files? 

/cc @apb7